src/DatePicker/DatePickerDialog.js

```diff
@@ -136,7 +138,7 @@ class DatePickerDialog extends Component {
           repositionOnUpdate={true}
           open={open}
           onRequestClose={this.handleRequestClose}
-          style={styles.dialogBodyContent}
+          style={Object.assign(styles.dialogBodyContent, containerStyle)}
```

Comment:
The extra `<div>` wrapping `Popover` came directly from `DatePickerInline`, and Popover doesn't position correctly without it, but the extra spread seems to be just copy-pasta, it should probably have been removed.